auth-web/src/views/auth/home/HomeView.vue

```diff
@@ -9,19 +9,11 @@
           <div class="hero-banner__cta-btns mb-2">
             <!-- Authenticated -->
             <div v-if="userProfile" class="cta-btns-authenticated">
-              <name-request-button :isWide="true" />
-              <v-btn large dark color="bcgovblue" class="cta-btn-auth font-weight-bold"
+              <v-btn large color="#003366" class="cta-btn-auth font-weight-bold white--text mr-4"
```

Comment:
Use a named color here.

It's already declared in the Vuetify theme: https://github.com/bcgov/sbc-auth/blob/5033b678f797002aa6178bd738ad0757d20be53e/auth-web/src/plugins/vuetify.ts#L33